src/SamplesApp/Directory.Build.targets

```diff
@@ -3,15 +3,6 @@
 	<!-- https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#use-case-multi-level-merging -->
 	<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
 
-	<!-- Ensure that Uno.Samples.UITest.Generator build before all SamplesApp project -->
-	<ItemGroup Condition="'$(MSBuildProjectName)'!='Uno.Samples.UITest.Generator' and '$(DocsGeneration)'==''">
-		<ProjectReference Include="$(MSBuildThisFileDirectory)\SamplesApp.UITests.Generator\Uno.Samples.UITest.Generator.csproj">
-			<ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-			<SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
-			<UndefineProperties>TargetFramework;RuntimeIdentifier;TargetFrameworks;RuntimeIdentifiers</UndefineProperties>
```

Comment:
just a question: Why is this line no longer necessary?

Reply:
The generator is now referenced in SamplesApp.UnitTests.targets. For the `UndefineProperties` line specifically, I'm not sure why it was required before, but it's probably for the days when we supported Xamarin (MonoAndroid12.0, etc) where referencing the generators had some unknown issues that I hadn't dug into at that time.

Currently, referencing the generator in Directory.Build.targets is very problematic when building Wasm with AOT enabled as this project references flows into a generated `aot-instances.csproj` where it shouldn't.

Reply:
Thanks!